sdk/cosmos/azure-cosmos-spark_3_2-12/src/test/scala/com/azure/cosmos/spark/SparkE2EChangeFeedITest.scala

```diff
@@ -235,6 +239,13 @@ class SparkE2EChangeFeedITest
 
     validationDF
       .show(truncate = false)
+
+    assertMetrics(meterRegistry, "cosmos.client.op.latency", expectedToFind = true)
+    assertMetrics(meterRegistry, "cosmos.client.system.avgCpuLoad", expectedToFind = true)
+    assertMetrics(meterRegistry, "cosmos.client.req.gw", expectedToFind = true)
+    assertMetrics(meterRegistry, "cosmos.client.req.rntbd", expectedToFind = true)
+    assertMetrics(meterRegistry, "cosmos.client.rntbd", expectedToFind = true)
```

Comment:
I wonder whether we should use rntbd in the metrics name -- should it be about directTcp/tcp or something

Reply:
My preference is to keep rntbd. It is shorter. It is the propriatary protocol we use and that shows up in diagnostics, class names etc. And I don't think directTcp means for customers it is easier to understand what it does. tcp is too confusing. Http uses TCP after all as well.